resources/metadata/mtaBuild.yaml

```diff
@@ -125,6 +125,16 @@ spec:
             fields:
               - name: mtarFilePath
   containers:
-    - name: mta
-      image: devxci/mbtci
+    - image: ppiper/mta-archive-builder
```

Comment:
I need to ask again what it means to have multiple items in the list here. My understanding is that only the first image will be used to run in the Jenkins wrapper. @OliverNocon is there a potential issue here?

Reply:
Oh, ok we have mutually exclusive conditions, that should be fine

Reply:
Exactly, I tested the conditions with our test projects and it seemed to work fine.

Reply:
the first one where the condition matches will be chosen ...